### PR TITLE
Update tiup component cluster scale-in

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add a warning note

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7572
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-cluster-scale-in.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ tiup cluster scale-in <cluster-name> [flags]
 - Data type: `BOOLEAN`
 - This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
 
-> **Note:**
+> **Warning:**
 >
-> Because the forced removal of a TiKV node does not wait for data to be scheduled, removing more than one serving TiKV node is at the risk of data loss.
+> When using this option to forcibly remove TiKV or TiFlash nodes that are in service or are going offline, these nodes will be deleted directly without waiting for data to be scheduled. This imposes high risk of data loass. Therefore, you are advised to use this option on nodes that are down only.
```
Comment (TomShawn) [inline tiup/tiup-component-cluster-scale-in.md:50]:
```suggestion
> When you use this option to forcibly remove TiKV or TiFlash nodes that are in service or are going offline, these nodes will be deleted directly without waiting for data to be scheduled. This imposes a high risk of data loss. Therefore, it is recommended to use this option only on down nodes.
```

File: tiup/tiup-component-cluster-scale-in.md [hunk 2]
```diff
@@ -45,9 +45,9 @@ tiup cluster scale-in <cluster-name> [flags]
 - Data type: `BOOLEAN`
 - This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
 
-> **Note:**
+> **Warning:**
 >
-> Because the forced removal of a TiKV node does not wait for data to be scheduled, removing more than one serving TiKV node is at the risk of data loss.
+> When you use this option to forcibly remove TiKV or TiFlash nodes that are in service or are going offline, these nodes will be deleted directly without waiting for data to be scheduled. This imposes a high risk of data loss. Therefore, it is recommended to use this option only on down nodes.
```
Comment (AstroProfundis) [inline tiup/tiup-component-cluster-scale-in.md:50]:
```suggestion
> When you use this option to forcibly remove TiKV or TiFlash nodes that are in service or are pending offline, these nodes will be deleted immediately without waiting for data to be migrated. This imposes a high risk of data loss. Therefore, it is recommended to use this option only on down nodes.
```

Comment (AstroProfundis) [reply tiup/tiup-component-cluster-scale-in.md:50]: We use "Pending Offline" as status for those nodes that are in the offline process but not finished in `tiup cluster display`, so it's better to use the same phrase in doc.

**Comment by shichun-0415** [comment]
@TomShawn @AstroProfundis, PTAL

**Review by lilyjazz — APPROVED** [review]

**Review by AstroProfundis — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

**Pull request merged**